openshift/generate-manifests.sh

```diff
@@ -58,6 +58,12 @@ for container_name in "${!IMAGE_MAPPINGS[@]}"; do
   $YQ -i 'select(.kind == "Namespace").metadata.annotations += {"workload.openshift.io/allowed": "management"}' "$TMP_KUSTOMIZE_OUTPUT"
 done
 
+# Exclude ValidatingAdmissionPolicy and ValidatingAdmissionPolicyBinding
+# APIs from downstream build as these API are currently not enabled by default.
+# In the OCP 4.17 (K8s 1.30) they will be become available
+# and we will be able to remove this to enforce admission policies
```

Comment:
Reminder for self: create an issue to remove this after branch date so OCP 4.17 doesn't remove this.

Reply:
Created an issue here: https://issues.redhat.com/browse/OPRUN-3296